projects/lux/src/lib/input/input.component.ts

```diff
@@ -66,13 +67,15 @@ export class InputComponent implements OnInit, ControlValueAccessor, Validator {
       required: 'Required field.',
       min: 'Minimum value is $min.',
       max: 'Maximum value is $max.',
-      email: 'Format should match example@example.com.'
+      email: 'Format should match example@example.com.',
+      url: 'Format should match example.com.'
```

Comment:
Error message: We are going to require the protocol, right?
Then change to:  `url: 'Format should match a full URI: https://example.com.'`

Reply:
Updated error message: https://github.com/metadevpro/lux/pull/171/commits/46d0b6f90168b069cf677e51ef03e4e7562dc55d
Can create a new method to check whether a URL is valid and includes the protocol if protocol is required